debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/YbExporterConsumer.java

```diff
@@ -82,6 +102,92 @@ void connect() throws URISyntaxException {
         flusherThread.start();
     }
 
+    /**
+     * Reads the lock file and checks if the PID is still running.
+     * If the PID is not running, it deletes the lock file.
+     * If the PID is running, it throws an IllegalStateException.
+     */
+    private void checkExistingLockFile() {
+        lockFile = new File(exportDir, String.format(".debezium_%s.lck", exporterRole));
```

Comment:
nit: 
i think this file should be created outside of this function. 
In acquire (lockFile.createNewFile();) , you sort of implicitly rely on this function being called. 

Better to create the File object outside in `connect()` is what I feel.